### PR TITLE
test: added units test for gtChars fieldValidator, closes #18

### LINKED ISSUE #18: Add unit tests to the *gtChars* validator for 100% coverage
Use the storybook and the `notNull` validator's tests as a guide, and add tests that mirror the stories.

Acceptance criteria:
* 100% test coverage of the validator.
* Style, etc. the same as the other unit tests.

### PR DESCRIPTION
**Pull request by evelyn-plee**

### CODE REVIEW

File: src/fieldValidators/string/gtChars/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Just, Left, Right, Nothing } from 'sanctuary'
+
+import gtChars from './'
+
+describe('fieldValidators:string:gtChars', () => {
+  it(`Left ({"failures": [{"errorType": "TOO_FEW_CHARACTERS", "testValue": Just(3)}], "value": Just ("Bob")})`, () => {
```
Comment (chasm) [inline src/fieldValidators/string/gtChars/index.spec.js]: I thought that I'd commented on this, but it appears that I didn't save it. We decided that these descriptions are too wordy and too difficult to understand (too technical). So we've changed them to this:

`Returns Left(T00_FEW_CHARACTERS error) when the length of the string is the same as the testValue`

`Returns Right(Just(value)) when the length of the string is greater than the testValue`

`Returns Right(Nothing) when the value is Nothing`

`Returns Right(Just(value)) when the testValue is Nothing`

Can you update them? Otherwise, perfect.

Comment (evelyn-plee) [reply src/fieldValidators/string/gtChars/index.spec.js]: Sounds good, I have updated them.
